application.js

```diff
@@ -35,11 +36,10 @@ app.use('/api', bodyParser.json({limit: '10mb'}));
  * For available stores, see
  * {@link https://github.com/feedhenry-raincatcher/raincatcher-user/tree/master/lib/session/mongoProvider.js}
  *
- * Redis is default store.
- * Switch to MongoDB:
- * 1. make sure to upgrade your database in Data Browser part of the RHMAP
- * 2. change 'store' attribute of sessionOptions to 'mongo'
- * 3. deploy
+ * Mongo is default store.
+ * It is expedted to get Mongo Connection Error on the first run on SAAS.
```

Comment:
This message should be more generic I think.

"This application requires a valid mongodb connection string to store session data."

The README should highlight the fact that the application requires the FH_MONGODB_CONN_URL environment variable to be a valid mongodb connection string.

It should also identify that the Data Browser in the RHMAP platform makes this environment variable available when the Data Browser has been updated